documentation/tables.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE IF NOT EXISTS `apikeys` (
```

Comment:
@mimicc83 Do you have a SQL dump for the original database? Please correct me, if the definitions are not fully correct so we can change them.